.github/workflows/cd.yml

```diff
@@ -23,14 +23,17 @@ on:
 jobs:
   cd:
     name: CD
-    uses: grafana/plugin-ci-workflows/.github/workflows/cd.yml@ddc6565117d4fd559e0f22c4f9cf1ac76c229e14 # ci-cd-workflows/v3.2.0
+    uses: grafana/plugin-ci-workflows/.github/workflows/cd.yml@ci-cd-workflows/v4.0.0 # ci-cd-workflows/v4.0.0
     with:
       branch: ${{ github.event.inputs.branch }}
       environment: ${{ github.event.inputs.environment }}
       docs-only: ${{ fromJSON(github.event.inputs.docs-only) }}
       attestation: true
       run-playwright: false
 
+      # Explicitly set Node.js version to 22 to ensure compatibility with yargs@18.0.0
+      node-version: '22'
```

Comment:
we should update [.nvmrc](https://github.com/grafana/logs-drilldown/blob/main/.nvmrc) to match